prow/config.yaml

```diff
@@ -615,3 +615,21 @@ presets:
   volumeMounts:
   - name: docker-graph
     mountPath: /docker-graph
+# volume mounts for kind
+- labels:
+    preset-kind-volume-mounts: "true"
+  volumeMounts:
+    - mountPath: /lib/modules
+      name: modules
+      readOnly: true
+    - mountPath: /sys/fs/cgroup
+      name: cgroup
+  volumes:
+    - name: modules
+      hostPath:
+        path: /lib/modules
+        type: Directory
+    - name: cgroup
+      hostPath:
+        path: /sys/fs/cgroup
+        type: Directory
```

Comment:
@BenTheElder @figo 
why not add the resource caps too:
```
      resources:
        requests:
          # these are both a bit below peak usage during build
          # this is mostly for building kubernetes
          memory: "9000Mi"
          # during the tests more like 3-20m is used
          cpu: 2000m
```
https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kind.yaml#L66-L72

Reply:
we should not add 9000mi to every job. I don't think this is building kubernetes

Reply:
also resource requests don't work in presets IIRC?

Reply:
i don't know...we probably shouldn't.